### ISSUE TITLE
[@types/tabulator-tables] Can't publish tabulator-types definitions with npm package.

### ISSUE THREAD

**Issue by zaniitiin** [DefinitelyTyped/DefinitelyTyped#51385]
@types/tabulator-tables is uses namespace to declare types definitions for tabulator-tables, Due to this we are not able to publish the type definitions with our internal npm package.

Our internal npm package contain a wrapper component for [Tabulator Tables](http://tabulator.info) for uniform style across multiple applications and platforms. We want to publish the tabulator definitions with our package. and to avoid an addition package in dependencies for consuming applications.
Although declaring namespace and class is not recommended anymore by typescripts itself, and declare definition statement don't allow to import to definitions.

- Authors: @Jojoshua

**Comment by Jojoshua** [comment]
Not sure if your issue is with modules. If so, read through some previous discussion.  https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44407

You could create something like this https://github.com/Jojoshua/TypedTabulator/blob/master/src/index.d.ts

**Comment by zaniitiin** [comment]
Yes, this works, but I'm looking a way to be able to ship this with my module package.
Scenario:
- I have a project that creates a wrapper component for Tabulator-tables, and I wanna publish this as a npm module.
Since I'm using typescript, I'm also using @types/tabulator-tables to have tabulator-types, and I'm declaring module same way as you mentioned in the above comment.

- Now when I wanna publish my component, I want have the type definition include with my component, and excluding an additional dependency on consuming app.

So I'm trying to finding a way to do so.
